specification/desktopvirtualization/DesktopVirtualization.Management/SessionHostManagement.tsp

```diff
@@ -59,6 +61,20 @@ interface ControlSessionHostUpdate {
   >;
 }
 
+@added(Versions.v2025_04_01_preview)
+@armResourceOperations
+@tag("SessionHostManagement")
+interface ControlSessionHostProvisioning {
+  /**
+   * Control provisioning of a hostpool.
```

Comment:
>hostpool [](http://example.com/codeflow?start=31&length=8)

Suggest explaining a bit more, like that it allows canceling host pool provisioning.